readthedocs/organizations/querysets.py

```diff
@@ -15,13 +15,29 @@ class BaseOrganizationQuerySet(models.QuerySet):
 
     """Organizations queryset."""
 
+    def public(self, user):
+        """
+        Return all organizations the user has access to.
+
+        If ``ALLOW_PRIVATE_REPOS`` is `False`, all organizations are public by default.
```

Comment:
I think we should probably rename this to `RTD_CORPORATE` or something. We have been using this setting to differentiate if we are on .org or .com lately, but it's not what the name suggests.